api.bs

```diff
@@ -921,6 +932,120 @@ To <dfn>deduct privacy budget</dfn> given a [=privacy budget key=] |key|,
 1.  Return whether |newValue| is greater than or equal to 0.
 
 
+### Epoch Start Store ### {#s-epoch-start}
+
+An [=epoch=] starts at a randomly-selected time
+for each [=site=].
+This ensures that any bias in aggregates
+that arises from a [=privacy budget=] reaching zero
+is averaged across all the contributions from all [=users=].
+
+The <dfn>epoch start store</dfn> is a [=map=] keyed by [=site=]
+with values that are [=moments=].
+
+To <dfn>get the current epoch</dfn>
+given an [=environment settings object=] |settings|,
+a [=site=] |site|,
+and [=moment=] |now|:
+
+1.  Let |period| be the [=duration=] of one [=epoch=] (i.e., 86400 seconds).
+
+1.  If the [=epoch start store=] does not [=map/contain=] |site|, [=map/set=]
+    its value to |now|, minus a [=duration=]
+    that is randomly selected
+    from between 0 (inclusive) and |period| (exclusive).
+
+1.  Let |start| be the result of [=map/get|getting the value=] of |site|
+    from the [=epoch start store=].
+
+1.  Let |elapsed| be |now| - |start| / |period|.
```

Comment:
Should this be `(|now| - |start|) / |period|`, i.e. subtraction before division?

Reply:
Of course it should.